game/graphics/opengl_renderer/Shader.cpp

```diff
@@ -64,6 +67,23 @@ Shader::Shader(const std::string& shader_name, GameVersion version) : m_name(sha
     return;
   }
 
+  // uniform samplers must be named matching the texture unit
+  glUseProgram(m_program);
+  for (int i = 1; i < 30; ++i) {
+    std::string uniformName = "tex_T" + std::to_string(i);
+    GLint texLoc = glGetUniformLocation(m_program, uniformName.c_str());
+    if (texLoc != -1) {
+      lg::debug("Shader {}: setting up texture uniform {}", shader_name, uniformName);
+    }
+    glUniform1i(texLoc, i);
+  }
+  // assuming that the bones uniform block is always using binding point 1
+  GLint bonesLoc = glGetUniformBlockIndex(m_program, "ub_bones");
+  if (bonesLoc != -1) {
+    lg::debug("Shader {}: setting up bones uniform block", shader_name);
```

Comment:
I would probably remove these two `lg::debug` statements, there's going to be a lot of them for all the shaders and they aren't that useful.

Reply:
There are only two shaders that have `ub_bones` uniforms, 23 shaders use some `tex_TXX` uniform, they would log more (49 total). I will remove the extra debug logs though, if they are not useful.